src/components/grid/Grid.tsx

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import { useEffect, useState } from 'react';
+
+import { Board, SquareTypes, Coordinates } from './SquareTypes';
+import HitboxUnavailable from './HitboxUnavailable';
+import Hitbox from './Hitbox';
+import Wall from './Wall';
+
+type PropsGrid = {
+  board: Board;
+  ws: WebSocket;
+  yourTurn: boolean;
+};
+const Grid: React.FC<PropsGrid> = ({ board, ws, yourTurn }) => {
+  const [grid, setGrid] = useState([<mesh></mesh>]);
+
+  const keys = Object.keys(board).map((key) => Number(key));
+
+  useEffect(() => {
+    const gridNew = [<mesh></mesh>];
+    let indexRow = 0;
```

Comment:
No need for variables like this. The map function in JS has index as the second parameter